src/table.ts

```diff
@@ -882,7 +885,7 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       const toRowStream = new Transform({
         transform: (rowData, _, next) => {
           if (
-            chunkTransformer._destroyed ||
+            chunkTransformer.canceled ||
```

Comment:
It seems like there are only 2 things that care about the cancelled flag:
- the closure for userstream.end()
- the closure for toRowStream

ChunkTransformer seems like an innocent bystander here. Why not move the userCancelled flag as a local var in createReadStream(), whose scope is shared by the 2 closure that care about the flag?

Reply:
Moving the flag closer to the consumer makes sense to me, I'll try to do it. I only put it here to replace the `_destroyed` logic, but you are right, it will make more sense downstream.

Reply:
Done